tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+
+from models import Base
+
+from .factories import AnimeFactory
+
+
+@pytest.fixture()
+def app():
+
+    db = SQLAlchemy(model_class=Base)
+
+    # create the app
+    app = Flask(__name__)
+
+    app.config.update({
+        "TESTING": True,
+    })
+
+    # configure the SQLite database for testing
+    app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///animeDB_test.db"
+
+    # initialize the app with the extension
+    db.init_app(app)
+
+    with app.app_context():
+
+        db.create_all()
+
+        db.session.add(AnimeFactory())
```

Comment:
Here AnimeFactory should do the job for you.
The point of factory_boy is to reduce the amount of code used to create fake data.

Reply:
I wonder if we could do simple here instead of specifically calling db.session.add

Reply:
I have already concern about this point. I tried to reduce the code by using this  [class](https://factoryboy.readthedocs.io/en/stable/orms.html#factory.alchemy.SQLAlchemyModelFactory) from Factory Boy, but I could not figure how to pass the db session to AnimeFactory in factories.py. Note that app fixture yield app, while db object should be declared in the app fixture (function) according to flask sqlalchemy docs. Though, I will try to work on it.